src/NServiceBus.Persistence.AzureTable/NServiceBus.Persistence.AzureTable.csproj

```diff
@@ -9,14 +9,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Data.Tables" Version="[12.8.3, 13.0.0)" />
-    <PackageReference Include="Newtonsoft.Json" Version="[13.0.3, 14.0.0)" />
-    <PackageReference Include="NServiceBus" Version="[9.0.0, 10.0.0)" />
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
```

Comment:
I couldn't bump this to latest because 12.9.1 has transitive CVEs

Reply:
Don't forget that we can add new top-level references to fix that!

Reply:
I had the hope they will patch it soon, but you are right!